src/timer.h

```diff
@@ -4,7 +4,8 @@
 #include "cm_quantile.h"
 
 typedef struct {
-    uint64_t count;     // Count of items
+    uint16_t actual_count; // Actual items recieved
```

Comment:
Lets make this an `uint64_t` since it is pretty easy to overflow a `uint16_t` within statsite